test/agent_helper.rb

```diff
@@ -279,7 +279,7 @@ def assert_metrics_recorded_exclusive(expected, options = {})
   assert_equal(0, unexpected_metrics.size, "Found unexpected metrics: #{format_metric_spec_list(unexpected_metrics)}")
 end
 
-def assert_newrelic_metdata_present(metadata)
+def assert_newrelic_metadata_present(metadata)
```

Comment:
I'm hoping tests don't care

Reply:
The tests that call this custom assertion were updated too, so everything should be fine!